NBitcoin.TestFramework/WellknownNodeDownloadData.cs

```diff
@@ -318,6 +314,39 @@ public class PolisNodeDownloadData
 			};
 		}
 
+		/// <summary>
+		/// Currently this is using the c++ stratisx full node. Only the linux version is available at this time.
+		/// Should eventually be changed to use C# full node once RPC interfaces are more mature.
```

Comment:
Yeah you probably want to use the fullnode, also because we have more RPC coverage on the C# node.